Esri/arcgis-maps-sdk-kotlin-samples
# Project Geometry

## Description
PR to add a new Kotlin sample _"Project Geometry"_ in `Geometry` category. This sample projects a point from one spatial reference to another.

## Links and Data
Sample Epic: `runtime/kotlin/issues/1123`

## What To Review
-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files


## How to Test
- Run the sample on the sample viewer or the repo. 
- Tap on the map once loaded, bottom Textview should display the coordinates in current spatial reference and in the new transformed spatial reference.
